styles/mobile.css

```
@@ -0,0 +1,52 @@
+body {
+    width: 100%;
+}
+
+.cabecalho {
+    background-color: var(--green);      
+    margin: 0;
+    width: 100%;
+}
+
+.titulo-principal {
+    color: var(--grey);
+    font-family: var(--primary-font);
+    font-size: 6rem;
+    padding: 1rem;
+    text-align: center;
+}
+
+.imagem-perfil {
+    border-radius: 4.938rem;
+    width: 100%;
+}
+
+.secao {
+    align-items: center;
+    display:inline-block;
+    margin-bottom: 1.5rem;
+}
+
+.lista {
+    display: inline-block;
+    gap: 5rem;
+    justify-content: center;
+}
+.item-lista {
+    align-items: center;
+    color: var(--black);
+    display: block;
+    font-size: 2rem;
+    justify-content: center;
+    list-style: none;
+}
+
+.link {
+    align-items: center;
+    color: var(--black);   
+    display: inline-flex;
+    font: var(--primary-font);
+    gap: 2rem;
+    margin-bottom: 2rem;
```

Mesma pergunta aqui. E se for remover o flex, algumas propriedades deixam de funcionar
```suggestion
    display: inline-flex;
    margin-bottom: 2rem;
```